example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/Stepdefs.java

```diff
@@ -9,17 +11,20 @@
 public class Stepdefs {
 	
 	private Belly belly = new Belly();
-	
+
+	@TestCaseId(isParameterized = true)
     @Given("^I have (\\d+) cukes in my belly$")
-    public void I_have_cukes_in_my_belly(int cukes) {
+    public void I_have_cukes_in_my_belly(@TestCaseIdKey(isInteger = true) int cukes) {
```

Comment:
update examples according to the new client changes

Reply:
updated